src/components/CreateProjectButton/CreateProjectButton.tsx

```diff
@@ -2,7 +2,11 @@ import { PlusIcon } from '@camp/design';
 import { Button } from '@mantine/core';
 import { Link } from '@tanstack/react-location';
 
-export const CreateProjectButton = () => {
+interface Props {
+  onClick?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
do we need this typing? 
if not we can just make it a void function just like u did with the [dismiss button](https://github.com/fullstacksjs/the-charity-frontend/blob/c9baf464163c55a1d6b57051b09c52b0e80a0aae/src/components/CreateProject/CreateProjectForm.tsx#L8) 

```suggestion
  onClick?: () => void;
```

Reply:
in my opinion, HTML event handers are not like `()=>void` so I defined it like this